lib/spoom/sorbet/metrics.rb

```diff
@@ -1,90 +1,32 @@
 # typed: strict
 # frozen_string_literal: true
 
+require_relative "sigils"
+
 module Spoom
   module Sorbet
-    class Metrics < T::Struct
+    module Metrics
```

Comment:
If the only responsibility is to parse the metrics output, how about:
```suggestion
    module MetricsParser
```